test/isObject.spec.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-env mocha */
+/* eslint-disable no-unused-expressions */
+
+const expect = require('chai').expect
+const { isObject } = require('../')
+
+const inputs = {
+  obj: {},
+  arr: [],
```

Comment:
@tbrandle would probably be worth testing a non-empty object & array to verify both work. Might also be worth testing additional variable types (integer, null, undefined, etc) since the signature should allow anything to be checked and return false if not exist or not an object